src/main.cpp

```diff
@@ -963,6 +961,12 @@ int main ( int argc, char** argv )
                 // show dialog
                 ClientDlg.show();
 
+                // Connect on startup if requested
+                if ( !strConnOnStartupAddress.isEmpty() )
+                {
+                    Client.Connect ( strConnOnStartupAddress, strConnOnStartupAddress );
+                }
+
```

Comment:
This is not nice, but it seems to crash the app if I factor out the if and pApp->exec() after the else

Reply:
Maybe we could introduce a method or signal which informs the client that the object is allocated and the connection can be established. Then the emits in the connect method would become redundant.